disttask/framework/handle/handle.go

```diff
@@ -90,6 +89,9 @@ func WaitGlobalTask(ctx context.Context, globalTask *proto.Task) error {
 			case proto.TaskStateReverted:
 				logutil.BgLogger().Error("global task reverted", zap.Int64("task-id", globalTask.ID), zap.Error(found.Error))
 				return found.Error
+			case proto.TaskStatePaused:
```

Comment:
If task in paused status，will the log print too much due to dispatche manger period check？

Reply:
> If task in paused status，will the log print too much due to dispatche manger period check？

No, dispatcher exits.